packages/template/package.json

```diff
@@ -7,7 +7,9 @@
   "scripts": {
     "dev": "vite",
     "build": "vite build",
-    "serve": "vite preview"
+    "serve": "vite preview",
+    "postinstall": "node postinstall.js",
+    "prepare": "node prepare.js"
```

Comment:
I am not entirely sure but it [seems](https://docs.npmjs.com/cli/v8/using-npm/scripts#prepare-and-prepublish) that the one to use might be `prepublishOnly`. It looks like the `prepare` hook is run for some other cases; Maybe these cases are not relevant in our case but I wanted to comment on it, just in case.